test/workloads.bats

```diff
@@ -25,7 +30,26 @@ resources = { "cpushares" =  "$cpushares", "cpuset" = "$cpuset" }
 EOF
 }
 
-function check_cpu_fields() {
+function setup_pod_tests() {
+	if [[ "$CONTAINER_CGROUP_MANAGER" == "cgroupfs" ]]; then
```

Comment:
@giuseppe @mrunalp @kolyshkin is there a nice way to create a slice with systemctl on the cmd line? docs didn't turn anything up